src/components/Container/Container.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 
-const Container = ({ children, ...props }) => (
+const Container = ({ children, isFreeUser, ...props }) => (
```

Comment:
rename this prop to `compact`. Use intent, not the use-case. 

`compact` should use less spacing if set.